### PR TITLE
[zh-cn]: update the translation of `<dir>` element

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

**Comment by T34-active** [comment]
@jasonren0403 PTAL

File: files/zh-cn/web/html/element/dir/index.md [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
 ---
 
-{{deprecated_header}}
+{{HTMLSidebar}}{{Deprecated_Header}}
 
-已废弃的 **HTML 目录元素**（**`<dir>`**）被作为一个文件和/或文件夹的目录的容器，可能还有 {{Glossary("user agent", "用户代理")}} 应用的样式与图标。Do not use this obsolete element; instead, you should use the {{HTMLElement("ul")}} element for lists, including lists of files.
+**`<dir>`** [HTML](/zh-CN/docs/Web/HTML) 元素用于作为包含文件或文件夹目录的容器，可能由{{Glossary("user agent", "用户代理")}}应用样式和图标。不应使用此过时元素；相反，对于包括文件列表在内的列表应使用 {{HTMLElement("ul")}} 元素。
 
-> **备注：** 不要使用这个元素。虽然它出现在早期的 HTML 规范中，它已经在 HTML4 中废除了，并且在 [HTML5](/zh-CN/docs/Web/Guide/HTML/HTML5) 中过时。使用 {{HTMLElement("ul")}} 来代替。
+> **警告：** 请不要使用这个元素。尽管它曾在早期 HTML 规范中出现，但在 HTML4 中已被弃用，并已完全移除。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/dir/index.md:12]:
```suggestion
> **警告：** 请不要使用这个元素。尽管它曾在早期 HTML 规范中出现，但在 HTML4 中已被弃用，并且已被完全移除。
```

File: files/zh-cn/web/html/element/dir/index.md [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
 
-- `compact`
+- `compact` {{Deprecated_Inline}}
+  - : 此布尔属性提示列表应以紧凑样式渲染。对该属性的解释取决于用户代理，且并不适用于所有浏览器。
 
-  - : 这个布尔属性表明，列表应该以紧凑样式显示。这个属性的实现取决于浏览器，并且不在所有浏览器中工作。
+## 规范
 
-    > **备注：** 用法注解：不要使用这个属性，因为它已经废除了： {{HTMLElement("dir")}} 元素应该使用 CSS 来设计。为了提供 `compact` 属性的相似效果，可以使用 [CSS](/zh-CN/docs/CSS) 属性 {{cssxref("line-height")}}，值为 `80%`。
+不属于任何规范。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/dir/index.md:27]:
```suggestion
不属于任何现行规范。
```

**Review by yin1999 — COMMENTED** [review]
nits:

**Comment by T34-active** [comment]
@yin1999 Thank you!

**Review by yin1999 — APPROVED** [review]
Thanks @T34-active

**Pull request merged**